core/controllers/editor.py

```diff
@@ -334,7 +332,7 @@ def get(self, exploration_id):
                 exploration_id, apply_draft=apply_draft, version=version))
```

Comment:
This is the editor file -- so what do you mean by "it's a part of playing exploration"?

Guest users do not have drafts; I'm talking about making sure the internal logic of _get_exploration_data() is safe here since it looks like apply_draft can get passed to it with a guest user.